vcl/extensions.vcl

```diff
@@ -11,7 +11,13 @@
 #
 
 sub hlx_type_pipeline_before {
-    set req.http.X-Dispatch-NoCache = "true";
+    # A non-production XFN header looks like: "repo--user.hlx.page"
+    # If you have an outer CDN, then it looks like "www.mydomain.com, repo--user.hlx.page"
+    # The filter below tests if you have an outer CDN and turns off the development mode
+    # caching (i.e. re-enables caching in helix-dispatch)
+    if(regsuball(regsuball(req.http.x-forwarded-host, "[^,]*(\.hlx(-[0-9]+)?\.page|\.project\-helix\.page|\.fastlydemo\.net)", ""), "[, ]", "") == "") {
+        set req.http.X-Dispatch-NoCache = "true";
```

Comment:
in the comment you say: it `re-enables` caching, but here you set `NoCache` to true ??

Reply:
yes, in any case, you set `NoCache` to true...

Reply:
Fixed the comments